EmailProcessing.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require 'public/include/init.php';
```

Comment:
Tu as remplacé require_once par require. Ce n'est pas du composer non plus. Il faut utiliser l'autoloader de composer.
Le fichier init.php, il fait quoi ?

Reply:
L'autoloader de composer sera mis en place avec l'issue "Add file required for routing"
Le fichier init.php, gére l'envoi de mail depuis le formulaire de contact.
Il comporte entre autre la clé secrete du recaptcha.

Reply:
Dans ton issue #5 tu mets ceci :
The description:
Folder / file tree construction
Restructuring of files in MVC architecture
Installation of composer and twig

Donc pour moi il est tout a fait normal de créer ton architecture MVC sur cette branche.